### PR TITLE
Create action-security.yml

### PR DESCRIPTION
**Pull request by sph4674**
Add New KBs #1265

### CODE REVIEW

**Comment by sph4674** [comment]
@h0x0er

File: knowledge-base/actions/johnwbyrd/update-release/action-security.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: 'Update Release'
+github-token:
+  action-input:
+    input: token
+    is-default: false
+  permissions:
+    contents: write
+    contents-reason: to get user input and update
```
Comment (h0x0er) [inline knowledge-base/actions/johnwbyrd/update-release/action-security.yml:8]: @sph4674 please change the reason to `to create tags and modify release`

File: knowledge-base/actions/johnwbyrd/update-release/action-security.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+    #Reference: https://github.com/johnwbyrd/update-release/blob/1d5ec4791e40507e5eca3b4dbf90f0b27e7e4979/src/main.ts
```
Comment (h0x0er) [inline knowledge-base/actions/johnwbyrd/update-release/action-security.yml:10]: Also update this reference link, you need to update it with the link having `line no.` on which `an api call` is made.
E.g:
* https://github.com/johnwbyrd/update-release/blob/1d5ec4791e40507e5eca3b4dbf90f0b27e7e4979/src/main.ts#L118
* https://github.com/johnwbyrd/update-release/blob/1d5ec4791e40507e5eca3b4dbf90f0b27e7e4979/src/main.ts#L220

**Comment by h0x0er** [comment]
@sph4674 please change the title of PR to something like `Add KB actions`

**Pull request closed without merging**